### PR TITLE
Vps 33/the invalid role back to scenarios button should be replaced

### PR DESCRIPTION
**Pull request by jacobmathew105**
## Describe the issue

The invalid role page contains a button that redirects the user to a redundant homepage when intuitively clicking through the button.

## Describe the solution

The “Back to scenarios” button on the Invalid Role page is replaced with a button saying “Try Access Scenario”. When Clicked, this button refreshes the page. The page contains the text: “Wait for your group member(s) to finish playing through their part of the scenario. Then, when it’s your turn to play through (your group members should let you know, click the below button: ”

Extra text as below is also added to the page: “If you have just finished playing your part of the scenario, let your group member with the {role with access} role know it’s their turn”.
image

## Risk

The page may reload indefinitely when the button is clicked. 

## Definition of Done

- [ ] Code peer-reviewed
- [ ] Wiki Documentation is written and up to date
- [ ] Unit tests written and passing
- [ ] Integration tests written and passing
- [ ] Continuous Integration build passing
- [ ] Acceptance criteria met
- [ ] Deployed to production environment

## Reviewed By

Who reviewed your PR - for commit history once merged

### CODE REVIEW

File: git status
Comment (bcho892) [inline git status:1]: needs to be deleted

**Comment by bcho892** [comment]
LGTM

**Review by harbassan — APPROVED** [review]
Yep everything's working nicely 👍

File: frontend/src/containers/pages/InvalidRolePage.jsx [hunk 1]
```diff
@@ -22,27 +22,32 @@ function InvalidRolePage({ group }) {
     fontWeight: "600",
     display: "flex",
     flexDirection: "column",
+    height: "100vh",
```
Comment (wjin-lee) [inline frontend/src/containers/pages/InvalidRolePage.jsx:25]: **Not an issue**, but on mobile, it's usually advised to consider using `dvh` **if** this was in any way targetted for mobile.

**Review by wjin-lee — APPROVED** [review]
LGTM!

**Pull request merged**